packages/editor/CodeMirror/createEditor.ts

```diff
@@ -253,6 +254,8 @@ const createEditor = (
 
 				// Apply styles to entire lines (block-display decorations)
 				decoratorExtension,
+				dropCursor(),
```

Comment:
[`dropCursor`](https://codemirror.net/docs/ref/#view.dropCursor) shows a cursor at the location where content will be dropped.